packages/devtools_app/lib/src/scaffold.dart

```diff
@@ -326,6 +326,29 @@ class DevToolsScaffoldState extends State<DevToolsScaffold>
       ],
     );
     final theme = Theme.of(context);
+    final scaffold = Scaffold(
```

Comment:
instead of breaking this out into its own var, put back inline in the return statement and see my comment below regarding KeyboardShortcuts. Nesting widgets directly is preferred for build methods when possible. There are some cases where we need to break widgets out into local variables, but in this case, the var `scaffold` is only being used once, so we can leave directly in the widget tree.

Reply:
Sg, done!